config.cfg.template

```diff
@@ -49,6 +49,9 @@ config = {
         # maximum number of items in single koji multicall. Too low values may
         # cause poor performance, too high values may cause timeouts.
         "multicall_chunk_size": 100,
+        # run scratch-builds from latest known repo_id to reduce hazard.
```

Comment:
Package dependencies can be resolvable by Koschei, but before build is submitted to Koji, they may break. In such case package would be marked as failing instead of unresolved.